app/src/pages/loginPage/loginPage.tsx

```diff
@@ -70,44 +70,47 @@ export const LoginPage = (props: ILoginPageProps): React.ReactElement => {
 
   return (
     <ContainingView>
-      <PaddingView paddingTop={PaddingSize.Wide}>
-        <Box variant='card' isFullWidth={false}>
-          <Form onFormSubmitted={onLoginClicked} isLoading={isLoading}>
-            <Stack isFullHeight={true} isFullWidth={true} shouldAddGutters={true} direction={Direction.Vertical} childAlignment={Alignment.Center}>
-              <Text variant='header3'>Welcome Back :)</Text>
-              <Spacing variant={PaddingSize.Wide} />
-              <SingleLineInput
-                inputWrapperVariant={emailError ? 'error' : undefined}
-                messageText={emailError || undefined}
-                placeholderText='Email Address'
-                inputType={InputType.Email}
-                value={email}
-                onValueChanged={onEmailTyped}
-              />
-              <Spacing variant={PaddingSize.Wide} />
-              <SingleLineInput
-                inputWrapperVariant={passwordError ? 'error' : undefined}
-                messageText={passwordError || undefined}
-                placeholderText='Password'
-                inputType={InputType.Password}
-                value={password}
-                onValueChanged={onPasswordTyped}
-              />
-              <Spacing variant={PaddingSize.Narrow} />
-              <Stack direction={Direction.Horizontal} shouldAddGutters={true} childAlignment={Alignment.Center}>
-                <Stack.Item growthFactor={1} shrinkFactor={1} />
-                <Link shouldOpenSameTab={true} text='Forgot Password?' target='/forgot-password' />
+      <ResponsiveContainingView sizeResponsive={{ base: 12, small: 8, medium: 6, large: 5 }}>
+        <Stack direction={Direction.Vertical} paddingVertical={PaddingSize.Wide2} isFullHeight={true}>
```

Comment:
i don't fully understand how to structure these yet. maybe chat about this over the weekend lol

Reply:
ye that would be great, need to make it easier to use!